src/alfasim_sdk/_internal/alfacase/case_description_attributes.py

```diff
@@ -192,6 +192,11 @@ class Foo:
         )
         return Curve(image, domain)
     elif isinstance(value, Curve):
+        if value.GetLength() == 0:
+            return Curve(
+                image=Array([1.0], value.GetImage().unit),
+                domain=Array([1.0], value.GetDomain().unit),
+            )
```

Comment:
This is not good =/
We should not create data from thin air.

Reply:
Huh this is no good, why are we returning a curve with a single point using this arbitrary values? If somewhere else we are getting an error about the curve being empty, we should fix the error there, not just returning an arbitrary curve like this one.

Reply:
cc @Gualcar @BeneBr